Dockerfile

```diff
@@ -8,9 +8,20 @@ WORKDIR /entrypoint
 RUN ["go", "build", "."]
 
 
+FROM debian:bullseye-slim as etc-locale-gen
+
+RUN ["bash", "-exo", "pipefail", "-c", "export DEBIAN_FRONTEND=noninteractive; apt-get update; apt-get install --no-install-{recommends,suggests} -y locales; apt-get clean; rm -vrf /var/lib/apt/lists/*"]
+
+COPY icingaweb2 /usr/share/icingaweb2
+COPY icinga-L10n /usr/share/icinga-L10n
+COPY uncomment-locales.pl /
+
+RUN ["/uncomment-locales.pl", "/etc/locale.gen"]
+
+
 FROM debian:bullseye-slim
 
-RUN ["bash", "-exo", "pipefail", "-c", "export DEBIAN_FRONTEND=noninteractive; apt-get update; apt-get install --no-install-{recommends,suggests} -y apache2 ca-certificates libapache2-mod-php7.4 libldap-common locales-all php-{imagick,redis} php7.4-{bcmath,bz2,common,curl,dba,enchant,gd,gmp,imap,interbase,intl,json,ldap,mbstring,mysql,odbc,opcache,pgsql,pspell,readline,snmp,soap,sqlite3,sybase,tidy,xml,xmlrpc,xsl,zip}; apt-get clean; rm -vrf /var/lib/apt/lists/*"]
+RUN ["bash", "-exo", "pipefail", "-c", "export DEBIAN_FRONTEND=noninteractive; apt-get update; apt-get install --no-install-{recommends,suggests} -y apache2 ca-certificates libapache2-mod-php7.4 libldap-common locales php-{imagick,redis} php7.4-{bcmath,bz2,common,curl,gd,gmp,intl,json,ldap,mbstring,mysql,pgsql,snmp,sqlite3,tidy,xml,xmlrpc,xsl,zip}; apt-get clean; rm -vrf /var/lib/apt/lists/*"]
```

Comment:
Not 100%ly, but these seem kinda exotic (from IW2 POV) to me. After all I've written that set: I've just thrown all PHP stuff there for the case some mod would need this. See this as a cleanup. But as these are just 10M, you're free to choose:

* I shall just drop the second commit
* You merge and we'll see whether someone complains

Reply:
You're at least removing one [optional requirement for Director](https://icinga.com/docs/icinga-director/latest/doc/02-Installation/#optional-requirements) (sybase).

If you want this PR merged with me approving it, remove the second commit. Otherwise it needs review from someone more familiar with this. However, I doubt saving 10MB is worth going through potential "but this module needs that package" issues/PRs.